internal/storage/fake/bucket.go

```diff
@@ -26,7 +26,10 @@ import (
 	"strings"
 	"unicode/utf8"
 
-	"github.com/googlecloudplatform/gcsfuse/internal/gcloud/gcs"
+	"github.com/googlecloudplatform/gcsfuse/internal/storage"
+	bucket2 "github.com/googlecloudplatform/gcsfuse/internal/storage/bucket"
```

Comment:
can we use bucket here instead of bucket2 ?

Reply:
bucket name is used as variable name, that's why I kept bucket2